content/blog/tidymodels-fairness/index.Rmd

```diff
@@ -0,0 +1,62 @@
+
+The work of scoping Posit’s resources for fair machine learning, in large part, involved striking the right balance between tools in these categories and integrating them thoughtfully among our existing functionality. Rather than supporting as many fairness-oriented tools as possible, our goal is to best enable users of our tools to reason well about the fairness-relevant decisions they make throughout the modeling process.
+
+## Additions to tidymodels
+
+<!-- TODO: if a post on survival / roundup comes before this post, link "most recent set of releases" to that post here -->
+
+The most recent set of tidymodels releases include changes that provide support for assessment and critique using the tidymodels framework. 
+
+<!-- TODO: change the tidymodels.org urls to the merged versions -->
+
+The most yardstick release introduces [a tool to create fairness metrics](https://yardstick.tidymodels.org/reference/new_groupwise_metric.html) with the problem context in mind, as well as [some outputs of that tool](https://yardstick.tidymodels.org/reference/index.html#fairness-metrics) implementing common fairness metrics. For a higher-level introduction to the concept of a groupwise metric, we've also introduced a [new package vignette](https://yardstick.tidymodels.org/articles/grouping.html). To see those fairness metrics in action, see [this new article on tidymodels.org](https://deploy-preview-48--tidymodels-org.netlify.app/learn/work/fairness-detectors/), a case study using data about GPT detectors.
+
+The most recent tune release integrates support for those fairness metrics from yardstick, allowing users to evaluate fairness criteria across resamples. To demonstrate those features in context, we've added [another new article on tidymodels.org](https://deploy-preview-52--tidymodels-org.netlify.app/learn/work/fairness-readmission/), modeling hospital readmission for patients with Type I diabetes.
+
+Notably, we haven't introduced functionality to support mitigation. While a number of methods have proliferated over the years to finetune models to act more fairly with respect to some fairness criteria, each apply only in relatively niche applications with modest experimental results [@agarwal2018; @mittelstadt2023]. We believe that, in practice, the efforts of practitioners---and thus our efforts to support them---are better spent engaging with the sociotechnical context of a given modeling problem [@holstein2019].
```

Comment:
What do you think about softening this paragraph somewhat, like "For now, we believe that..."? This is fairly strong as is in terms of "we're not investing in mitigation" and I know that at least I personally would love for us to keep the options open.